javatests/com/google/copybara/git/GitEnvironmentTest.java

```diff
@@ -28,7 +28,7 @@
 public class GitEnvironmentTest {
 
   private static final Map<String, String> environment =
-      ImmutableMap.<String, String>builder().put("FOO", "123").put("BAR", "456").build();
+      ImmutableMap.<String, String>builder().put("FOO", "123").put("BAR", "456").put("LANG", "en_US.UTF-8").build();
 
```

Comment:
I think this is not testing the feature, right? Could you add a test case that fails without the fix and passes with it?

Thanks!!